WNPRC_EHR/resources/web/wnprc_ehr/wnprcReports.js

```diff
@@ -1329,7 +1329,7 @@ EHR.reports['abstract'] = function(panel, tab){
         });
 
         LABKEY.Utils.requiresCSS("wnprc_ehr/HousingAndAssignmentHistory.css");
-        WNPRC_EHR.Utils.Lib.loadLibrary(['/webutils/lib/webutils'], function() {
+        LABKEY.requiresScript('/webutils/lib/webutils', function() { 
```

Comment:
@dnicolalde reason for the change? i assume this report functions the same, though.

Reply:
Is a change on how webutils calls for dependencies.